tests/test_ports3_stride1.py

```diff
@@ -247,10 +245,11 @@ def test_ports3_stride1(read_delay=1,
         tester.step(2)
 
     with tempfile.TemporaryDirectory() as tempdir:
+        tempdir = "dumps"
```

Comment:
delete 248 and trace on 252?

Reply:
I think Max made that. I will delete it.